math/fplll/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        fplll fplll 5.4.3
+categories          math
+license             LGPL-2.1
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Lattice algorithms using floating-point arithmetic
+long_description    fplll contains implementations of several lattice algorithms. \
+                    The implementation relies on floating-point orthogonalization, \
+                    and LLLis central to the code, hence the name.
+checksums           rmd160  a56c2d79ff9fe12c74efe01311a05d35dbc3ed12 \
+                    sha256  c8c7532be9c1bfae4c85717a7dc37b68a678ac6713af6968e842738e508b5db3 \
+                    size    2335592
+
+depends_build-append \
+                    port:autoconf \
+                    port:automake \
+                    port:libtool \
+                    port:pkgconfig
+
+depends_lib-append  port:gmp \
+                    port:mpfr
+
+compiler.cxx_standard 2011
+
+# Temporary solution until non_lazy_ptr bug is fixed.
+compiler.blacklist-append \
+                    macports-gcc-12
+
+pre-configure {
```

Comment:
Another way to do this would be `use_autoconf yes`.  Instead of that whole `pre-configure` block, you can do  something like:

```
use_autoconf        yes
autoconf.cmd        ./autogen.sh
```

Reply:
@herbygillot Let me try it now, thank you.